### PR TITLE
[10.0][add] delivery_multi_destination

### PR DESCRIPTION
**Pull request by luismontalba**
Multiple destinations for the same delivery method
========================================

This module allows to set different price rules depending on the destination.

This module restores the concept of delivery grid, reusing the same model for nesting several "children" delivery methods, one per possible destination.
It has been designed to reuse all possible extensions to the base delivery, without the need to create a glue module for having multiple destinations.

Configuration
-----------------
To configure delivery methods with multiple destinations:

* Go to Inventory > Configuration > Delivery > Delivery Methods
* Create or edit an existing record.
* Select "Destination type" = "Multiple destinations".
* Introduce a line for each destination in the new tab "Destinations"
* Lines have priority, so you have to put first the lines with more restricted destinations.

Usage
--------
* When using the delivery method in a Sales order, delivery address will be used for computing the delivery price according introduced destinations.

cc @Tecnativa

### CODE REVIEW

File: delivery_multi_destination/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "delivery",
+    ],
+    "data": [
+        "views/delivery_carrier_view.xml",
+    ],
+    # "uninstall_hook": "uninstall_hook",
+    # "post_init_hook": "post_init_hook",
```
Comment (chienandalu) [inline delivery_multi_destination/__manifest__.py]: Can't you drop this lines?

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1:

File: delivery_multi_destination/__manifest__.py (hunk 1 above)
Comment (luismontalba) [reply delivery_multi_destination/__manifest__.py]: Done

**Review by rami-wafaie — APPROVED** [review]
Looks fine for me too 
:+1:

**Review by lasley — APPROVED** [review]
Thanks @luismontalba

**Comment by lasley** [comment]
Can you squash your two commits please?

**Comment by luismontalba** [comment]
@lasley , my two commits squashed

**Comment by lasley** [comment]
Thanks @luismontalba

**Pull request merged**